tests/parsers/test_neb.py

```diff
@@ -58,7 +58,6 @@ def test_neb_default(fixture_localhost, generate_calc_job_node, generate_parser,
 
     assert calcfunction.is_finished, calcfunction.exception
     assert calcfunction.is_finished_ok, calcfunction.exit_message
-    assert not orm.Log.objects.get_logs_for(node)
```

Comment:
Why remove these tests? This is making sure there were no error/warning logs

Reply:
Actually, with the new warning parsing there is a warning here for the line:

```
DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
```

Let me see if I can just get the errors tested here.

Reply:
This should do the trick:

```python
assert not [log for log in orm.Log.objects.get_logs_for(node) if log.levelname == 'ERROR']
```